test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -110,6 +110,26 @@ const getGlobalStylesUpdateEvents = async ( driver ) => {
 		( [ eventName ] ) => eventName === 'wpcom_block_editor_global_styles_update'
 	);
 };
+
+const saveGlobalStyles = async ( driver ) => {
+	await driverHelper.clickWhenClickable( driver, By.css( '.edit-site-save-button__button' ) );
+	const allCheckboxes = await driver.findElements(
+		By.css( '.entities-saved-states__panel .components-checkbox-control__input' )
+	);
+	allCheckboxes.forEach( async function ( element ) {
+		await element.click();
+	} );
```

Comment:
I don't think we can use `forEach` here as it would continue the execution without waiting for the clicks to happen (see https://stackoverflow.com/a/37576787/7225515). We should be alright with the `for` loop instead:
```suggestion
	for ( const checkbox of allCheckboxes ) {
		await driverHelper.setCheckbox( checkbox );
	}
```

Reply:
Hrm I committed this, but testing now im not sure it works here.  Even after adding the missing driver, this is set up to set a checkbox based on its selector, where instead we need to loop through all elements of the same selector (not selected A checkbox, but all possible checkboxes in the list).
I think using `await checkbox.click()` should still work though.

Reply:
Ah, sorry! I can see now what I've missed there. This should work:

```js
for ( const checkbox of allCheckboxes ) {
	await driverHelper.setCheckbox( driver, () => checkbox );
}
```

This will work because Selenium `locator` can also be a function that returns a `WebElement` instance.

Reply:
Oh neat!  That makes sense 😁 .  I updated to use that, (and added a 3rd arg for `false` since we need to deselect them all).

Right now, checked items are local to the panel so every time the panel is opened every checkbox is set `true` by default.  So while `checkbox.click()` on everything would work for now, being able to use `setCheckbox` makes this a bit more robust in case that behavior ever changes in the future. 🎉